### PR TITLE
Added GraphQL type descriptions

### LINKED ISSUE #1247: Explorer: GraphQL type descriptions
Our custom GraphQL types have no description.  We could provide short descriptions for schema introspection.

### PR DESCRIPTION
**Pull request by Akamig**
This PR resolves #1247, and "Tried but failed" to mediate #2065, but failed due to Generics of each queries in explorer.
Is there any good Idea to circumvent each IActions required by all queries?

### CODE REVIEW

**Review by longfin — CHANGES_REQUESTED** [review]
@Akamig Sadly, the build is failing. 😢there are two main reasons.

1. Lint is complaining "Line must be no longer than 100 characters"
2. `StandaloneSchema` needs a type argument `T`.
  - but... I don't have confidence about `StandaloneSchema` itself. (because I guess it's too ambigus naming)
   - how about opening another PR for #2065 ?

**Comment by Akamig** [comment]
@longfin 
Alright, I changed :
- Remove StandaloneQuery and Schema, which requires <T>.
- Shorten descriptions fit under 100 characters. Now it resolves only #1247.

**Review by longfin — CHANGES_REQUESTED** [review]
@Akamig Oops, lint is now complaining that "The parameter spans multiple lines". 😢 

I guess it would be better checking `dotnet build` on your local env before commit & push. 🙏

**Comment by Akamig** [comment]
...Which fails because of error SA1118: The parameter spans multiple lines. 
Maybe should I shorten description to fit under 100 characters? 
but It may confuse users because of lack of additional description.

**Comment by longfin** [comment]
> but It may confuse users because of lack of additional description.

It seems that make sense. then, disabling lint using `#progma warning` (like [this](https://github.com/planetarium/libplanet/blob/ff57748bd58351a4b01da3bdc9a497996ace8a1f/Libplanet.Net.Tests/BoundPeerTest.cs#L62)) can be solution. cc @planetarium/libplanet

**Review by longfin — DISMISSED** [review]

File: Libplanet.Explorer/GraphTypes/BlockType.cs [hunk 1]
```diff
@@ -13,45 +13,78 @@ public class BlockType<T> : ObjectGraphType<Block<T>>
     {
         public BlockType()
         {
+            #pragma warning disable SA1118
```
Comment (riemannulus) [inline Libplanet.Explorer/GraphTypes/BlockType.cs:16]: Is it necessary? if that. IMO we need more detailed comments about [SA1118](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1118.md).

Comment (Akamig) [reply Libplanet.Explorer/GraphTypes/BlockType.cs:16]: @riemannulus 
You mean we need some justification message under that pragma disable?
Is commenting out the reason `//like this` would be enough?

**Review by riemannulus — APPROVED** [review]

**Pull request merged**